lib/shared/messages.ts

```diff
@@ -84,8 +85,10 @@ export const warning = {
 		drive: { device: string; size: number },
 	) => {
 		return [
-			`This image recommends a ${image.recommendedDriveSize}`,
-			`bytes drive, however ${drive.device} is only ${drive.size} bytes.`,
+			`This image recommends a ${prettyBytes(image.recommendedDriveSize)}`,
+			`bytes drive, however ${drive.device} is only ${prettyBytes(
```

Comment:
Remove `bytes` here as the unit will be included in `prettyBytes(image.recommendedDriveSize)`